style.css

```diff
@@ -1,139 +1,648 @@
-#top-nav {
-  display: flex;
-  justify-content: space-between;
-  margin-top: 50px;
+@import url("https://fonts.googleapis.com/css2?family=Poppins:wght@500;600;700&display=swap");
+
+*,
+::before,
+::after {
+  box-sizing: border-box;
+  font-family: "Poppins", sans-serif;
 }
 
-#top-nav a {
-  text-decoration: none;
-  font-family: Poppins, sans-serif;
-  font-weight: bold;
-  font-size: 30px;
+body,
+h1,
+h2,
+h3,
+p {
+  margin: 0;
+  padding: 0;
+}
+
+body {
+  overflow-x: hidden;
+}
+
+.main_container {
+  background: #f7f7f9;
+  position: relative;
+}
+
+.background-white {
+  background: #fff;
+}
+
+.icon {
+  width: 8px;
+  height: 8px;
+}
+
+.bg-n-300 {
+  background: #f7f7f9;
+}
+
+.bg-p-50 {
+  background: #ebebff;
+}
+
+.clr-n-900 {
+  color: #000;
+}
+
+.clr-n-800 {
+  color: #172b4d;
+}
+
+.clr-n-700 {
+  color: #253858;
+}
+
+.clr-n-600 {
+  color: #344563;
+}
+
+.clr-n-500 {
+  color: #7a869a;
+}
+
+.clr-p-500 {
+  color: #396df2;
+}
+
+.clr-p-400 {
   color: #6070ff;
-  display: flex;
-  align-items: center;
-  text-align: center;
-  margin-left: 24px;
 }
 
-#top-nav a:hover {
-  text-decoration: underline;
-  text-decoration-color: #23a3fa;
-  text-decoration-thickness: 2px;
+.clr-p-300 {
+  color: #7f8cff;
+}
+
+.size-40 {
+  font-size: 40px;
+}
+
+.size-32 {
+  font-size: 32px;
+}
+
+.size-20 {
+  font-size: 20px;
+}
+
+.size-18 {
+  font-size: 18px;
+}
+
+.size-17 {
+  font-size: 17px;
+}
+
+.size-16 {
+  font-size: 16px;
+}
+
+.size-15 {
+  font-size: 15px;
+}
+
+.size-13 {
+  font-size: 13px;
+}
+
+.size-12 {
+  font-size: 12px;
 }
 
-#top-nav nav {
-  width: 45px;
-  height: 30px;
+.weight-700 {
+  font-weight: 700;
+}
+
+.weight-500 {
+  font-weight: 500;
+}
+
+.weight-400 {
+  font-weight: 400;
+}
+
+.line_hieght-52 {
+  line-height: 52px;
+}
+
+.line_hieght-28 {
+  line-height: 28px;
+}
+
+.line_hieght-24 {
+  line-height: 24px;
+}
+
+.line_hieght-20 {
+  line-height: 20px;
+}
+
+.line_hieght-16 {
+  line-height: 16px;
+}
+
+.link {
+  text-decoration: none;
+}
+
+.ul {
+  list-style: none;
+  margin: 0;
+  padding: 0;
+}
+
+.flex {
   display: flex;
+}
+
+.grid {
+  display: grid;
+}
+
+.btn:hover {
+  animation-name: btn-hover;
+  animation-fill-mode: forwards;
+  animation-delay: 0s;
+  background-color: rgb(101, 86, 235);
+}
```

Comment:
Great job 👍 on implementing the button hover effect! However🙆‍♀️, it does not match the design in Figma. Please 🙏update the text color and background color to match the Figma design. This will help your design resemble the awesome look of Figma.

## Your button hover
![image](https://user-images.githubusercontent.com/63445083/227600415-b478b571-261f-4d95-8763-b841de5921ca.png)

## Figma button hover
![image](https://user-images.githubusercontent.com/63445083/227600716-90c820ab-7f7c-4d2a-8344-5d26e1f96292.png)